Gemfile

```diff
@@ -65,6 +65,7 @@ gem 'cancancan'
 gem 'doorkeeper'
 
 # db
+gem 'active_record_union'
```

Comment:
Wow nice, hadn't heard of this before. This seems sick, and can definitely clean up a lot.

Reply:
Yeah, the other spot I noticed that had something similar was the game search, where we find game IDs from both games and game aliases.  It seemed useful for the long term for some more advanced things.